sdk/communication/AzureCommunicationChat/Source/ChatClient.swift

```diff
@@ -99,6 +99,52 @@ public class ChatClient {
         )
 
         self.service = client.chat
+        self.signalingClient = try createSignalingClient(credential: credential)
+    }
```

Comment:
Do you want client creation to fail if a signaling client cannot be created? What if they don't care about receiving realtime notifications? This seems like a significant change.